src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,68 @@
+import classnames from 'classnames';
+
+import { CommonProps, px } from '../../utils';
+import Subscribe from '../Subscribe/Subscribe';
+
+export interface FooterProps extends CommonProps {
+  /**
+   * Copyright data added to bottom of site
+   */
+  copyright: string;
+  /**
+   * Navigation items
+   */
+  navigation: React.ReactNode;
+  /**
+   * Social icons
+   */
+  socialIcons: React.ReactNode;
+  /**
+   * Header text for social icons
+   */
+  socialText?: string;
+  /**
+   * Subscribe blurb text
+   */
+  subscribeBlurb?: string;
+  /**
+   * Subscribe callback function
+   */
+  subscribeCallback: (e: React.MouseEvent) => void;
+  /**
+   * Subscribe title text
+   */
+  subscribeTitle?: string;
+}
+
+const Footer = ({
+  className,
+  copyright,
+  id,
+  navigation,
+  socialIcons,
+  socialText = 'Follow on Social',
+  subscribeBlurb = 'Receive exclusive content about our auctions, exhibitions, and special events.',
+  subscribeCallback,
+  subscribeTitle = 'Subscribe to Newsletter',
+}: FooterProps) => {
+  return (
+    <footer data-testid={id ? `footer-${id}` : `footer`} id={id} className={classnames(`${px}-footer`, { className })}>
```

Comment:
Pass through for any consumer of the component
```suggestion
    <footer {...props }data-testid={id ? `footer-${id}` : `footer`} id={id} className={classnames(`${px}-footer`, { className })}>
```